runtime/src/main/java/org/corfudb/protocols/wireprotocol/MultipleReadRequest.java

```diff
@@ -16,24 +14,26 @@
 @Data
 @AllArgsConstructor
 public class MultipleReadRequest implements ICorfuPayload<MultipleReadRequest> {
-    @Getter
+
+    // List of requested addresses to read.
     final List<Long> addresses;
 
+    // Whether the read results should be cached on server.
+    final boolean cacheableOnServer;
```

Comment:
Consider changing this veriable name to `cacheResponse ` or `cacheResult`

Reply:
Done.